src/LiberisLabs.CompaniesHouse.IntegrationTests/Tests/CompanyFilingHistoryTests/CompanyFilingHistoryTestsInvalid.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NUnit.Framework;
+
+namespace LiberisLabs.CompaniesHouse.IntegrationTests.Tests.CompanyFilingHistoryTests
+{
+    [TestFixture]
+    public class CompanyFilingHistoryTestsInvalid : CompanyFilingHistoryTestBase
+    {
+        private const string InvalidCompanyNumber = "ABC00000";
+
+        [SetUp]
+        protected override void When()
+        {
+            WhenRetrievingAnInvalidCompanyProfile();
+        }
+
+        [Test]
+        public void ThenTheProfileIsNotReturned()
+        {
+            Assert.That(_result.Data.Items, Is.Null);
```

Comment:
Let me know what you think with that information @kevbite. I can make it null if needed.